scanpy_scripts/cmd_options.py

```diff
@@ -978,6 +978,30 @@
         ),
     ],
 
+    'combat': [
+	*COMMON_OPTIONS['input'],
```

Comment:
Please use 4 white spaces as a unit of indentation. Python is quite unforgiving towards mixed use of spaces and tabs.